net/graphsync_fetcher_test.go

```diff
@@ -50,8 +51,8 @@ type notDecodable struct {
 }
 
 func init() {
-	cbor.RegisterCborType(notDecodable{})
-}
+	encoding.RegisterIpldCborType(notDecodable{})
```

Comment:
Why do we register some types inline in regular source files but register others using the code generating tool?

Reply:
the ones left behind are because they are doing non-standard things I slowly need to add support for